ipa-core/src/protocol/ipa_prf/oprf_padding/insecure.rs

```diff
@@ -8,6 +8,7 @@ use rand_core::{CryptoRng, RngCore};
 use crate::protocol::ipa_prf::oprf_padding::distributions::{
     BoxMuller, RoundedBoxMuller, TruncatedDoubleGeometric,
 };
+pub use crate::protocol::ipa_prf::oprf_padding::insecure::Error as DpError;
```

Comment:
I think `pub type DpError = Error` would be more appropriate here. Or, if you're referencing it by fully qualified path in `src/error.rs`, you could do away with this entirely. Having `DpError` as an alias for this `Error` makes more sense in other modules where plain `Error` could conflict.